server/testdriver/junit4/src/main/java/org/infinispan/server/test/junit4/InfinispanServerRuleBuilder.java

```diff
@@ -24,6 +25,18 @@
    private boolean jmx;
    private boolean parallelStartup = true;
 
+   /**
+    * Use this method to instantiate a single clustered embedded server
+    *
+    * @return InfinispanServerRule
+    */
+   public static InfinispanServerRule standalone() {
+      return new InfinispanServerRuleBuilder(DEFAULT_CLUSTERED_INFINISPAN_CONFIG_FILE_NAME)
```

Comment:
IMO we should add a constructor without any parameters that uses the default server configuration.

Reply:
s/standalone()/server()/

Reply:
oki